src/services/api/LocalStorageService.ts

```diff
@@ -0,0 +1,24 @@
+import { IClause } from "configurations/apiSchema";
+
+const DEALS_DATA = "DEALS_STORAGE_V1";
+const CLAUSES_DATA = "DEALS_STORAGE_V1";
+
+export class LocalStorageService {
+  public getClausesData(): IClause[] {
```

Comment:
I see your reasoning.

My idea was
1. `LocalStorageService` is just temporary (before it will get replaced with real API client)
2. `DataStorageService` will replace `LocalStorageService`
  1. So reduce as much dependencies as possible (this includes the local storage keys)
  2. Reduce type verbosity (`as IClause[]`) in `DataStorageService`, because API client returns the type for us. (will it? it should imo)

So from my perspective, having the an explicit method like `getClauseData` is okay, BUT it _is_ more work, true.